content/rest-api/index.textile

```diff
@@ -1002,8 +1002,9 @@ bc[sh]. curl https://rest.ably.io/push/channelSubscriptions \
 h5. Parameters
 
 - channel := Filter to restrict to subscriptions associated with that @channel@.
-- clientId := Filter to restrict to subscriptions associated with that @clientId@. Cannot be used with @deviceId@.
-- deviceId := Filter to restrict to subscriptions for that @deviceId@. Cannot be used with @clientId@.
+- clientId := Filter to restrict to subscriptions associated with that @clientId@. Cannot be used with @deviceId@, unless @concatFilters@ is set to @true@.
+- deviceId := Filter to restrict to subscriptions for that @deviceId@. Cannot be used with @clientId@, unless @concatFilters@ is set to @true@.
+- concatFilters := _false_ Find all device registrations which match either @clientId@ or @deviceId@. Set this to @true@ and set both @clientId@ and @deviceId@.
```

Comment:
Perhaps it's me misunderstanding the function being described, but I think this sentence needs fixing like this:

```suggestion
- concatFilters := _false_ Find all device registrations which match either @clientId@ or @deviceId@. Set this to @true@ in order to set both @clientId@ and @deviceId@.
```